CONTRIBUTING.md

```diff
@@ -157,6 +157,7 @@ git submodule update --init --remote --recursive
 - Create the conda development environment `cudf_dev`:
 ```bash
 # create the conda environment (assuming in base `cudf` directory)
+# note: `channel_priority` cannot be set to `strict`; use `flexible` instead
```

Comment:
What do you think about a rewording like this? 🙂

```suggestion
# note: RAPIDS currently doesn't support `channel_priority: strict`; use `channel_priority: flexible` instead
```

Reply:
Done.